resources/geocoding/hy/374.txt

```diff
@@ -81,8 +81,8 @@
 37424492|Փանիկ
 37424495|Արևշատ
 37424496|Մեծ Մանթաշ
-374245|Աշոցք
-374246|Ամասիա
+374245|Աշոցքի շրջան
+374246|Ամասիաի շրջան
```

Comment:
Why add "շրջան"? Is it to be consistent with the English use of "region" for these ranges?
Does the use of "շրջան" make a difference to a local person? How about using "Ashotsk region" instead of "Ashotsk" in English, would they mean different things? If not, maybe we remove "region" from the English file.